fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -564,7 +564,9 @@ protected void addResourceHttpHeaders(final FedoraResource resource) {
                     .build();
 
             servletResponse.addHeader(CONTENT_TYPE, binary.getMimeType());
-            servletResponse.addHeader(CONTENT_LENGTH, String.valueOf(binary.getContentSize()));
+            if (!binary.isRedirect()) {
```

Comment:
My only suggestion would be to add a comment here mentioning why we would want to drop the header for redirects